app/src/main/java/com/example/sharingang/DetailedItemFragment.kt

```diff
@@ -52,6 +54,7 @@ class DetailedItemFragment : Fragment() {
         }
 
         initiateWishlistButton(binding)
+        initRating(binding)
```

Comment:
Maybe better to make binding a `private lateinit var` attribute of the class because you need it in a few methods

Reply:
good call